Formula/screen.rb

```diff
@@ -54,11 +54,17 @@ def install
     # https://savannah.gnu.org/bugs/index.php?59465
     ENV.append "CFLAGS", "-Wno-implicit-function-declaration"
 
+    # master branch configure script has no 
+    # --enable-colors256, so don't use it
+    # when `brew install screen --HEAD`
+    args = []
```

Comment:
If you're creating an `args` array, why not just include all the other arguments to `configure` in it?

Reply:
Just seemed the least noisy PR. Can happily do:

```
args =  ["--prefix=#{prefix}"],
args << "--mandir=#{man}"
args << "--infodir=#{info}",
args <<  "--enable-pam" 
args << "--enable-colors256" unless build.head?

system "./configure", *args
```

if that's better stylistically